requirements.txt

```diff
@@ -6,6 +6,8 @@
 #
 apispec[yaml]==6.3.0
     # via Flask-AppBuilder (setup.py)
+authlib==1.2.1
```

Comment:
can you move this to requirements extra?

Reply:
I noticed that `authlib` is already in `requirements-extra.txt` - now I just upgraded the version there from `0.15.4` to the latest version `1.2.1`